public_html/wp-content/mu-plugins/4-helpers-wcpt.php

```diff
@@ -270,9 +279,20 @@ function get_wordcamp_dropdown( $name = 'wordcamp_id', $query_options = array(),
 				<?php
 
 				echo esc_html( $wordcamp->post_title );
+				$meta = '';
```

Comment:
```suggestion
				$start_date = '';
```

That seems more descriptive to me. At first I was confused because I thought this was meant to represent the full array, and thought there might be a bug until I slowed down and read the code closely.